src/Steeltoe.Management.EndpointBase/Middleware/EndpointMiddleware.cs

```diff
@@ -110,9 +111,10 @@ public virtual string HandleRequest(TRequest arg)
 
         public override bool RequestVerbAndPathMatch(string httpMethod, string requestPath)
         {
-            return _exactRequestPathMatching
-                ? requestPath.Equals(_endpoint.Path) && _allowedMethods.Any(m => m.Method.Equals(httpMethod))
-                : requestPath.StartsWith(_endpoint.Path) && _allowedMethods.Any(m => m.Method.Equals(httpMethod));
+            return (_exactRequestPathMatching ? requestPath.Equals(_endpoint.Path)
+               : requestPath.StartsWith(_endpoint.Path))
```

Comment:
Its nice that this change works as a one-liner, but having the ternary split across lines and combined with two other conditions makes it a bit harder to comprehend quickly, at least for me. I'd suggest formatting it so that either the ternary is on a single line, or split across 3 lines
```
return (_exactRequestPathMatching 
			? requestPath.Equals(_endpoint.Path) 
			: requestPath.StartsWith(_endpoint.Path))
	  && _endpoint.Enabled
	  && _allowedMethods.Any(m => m.Method.Equals(httpMethod));

	  
return (_exactRequestPathMatching ? requestPath.Equals(_endpoint.Path) : requestPath.StartsWith(_endpoint.Path))
	  && _endpoint.Enabled
	  && _allowedMethods.Any(m => m.Method.Equals(httpMethod));
```

Reply:
Ok makes sense. I like it split across 3 lines ... Ill make that change.